operations-guide/setting-up-axon-server/command-line.md

```diff
@@ -8,6 +8,7 @@ For the Axon Server Standard edition the only supported commands are:
 - users
 - register-user
 - delete-user
+- delete-all-events (only in [Development Mode](../../operations-guide/setting-up-axon-server/development-mode.md))
```

Comment:
Just a thought: if we call this operation "purge-events" (or just "purge"), we could also use this for the clustered mode to indicate that we want to delete the events of a removed context.

Reply:
I have another question: Will this only remove events from Axon Server Store? If yes, then what about tracking tokens that exist in our DBs, should we consider this as well, and write about it?

Reply:
Yes only the events are removed. I will add a note on that point to the docs